packages/create-instance/create-instance.js

```diff
@@ -60,20 +60,33 @@ export default function createInstance (
   addListeners(vm, options.listeners)
 
   if (options.scopedSlots) {
+    if (window.navigator.userAgent.match(/PhantomJS/i)) {
+      throwError('the scopedSlots option does not support strings in PhantomJS. Please use Puppeteer, or pass a component.')
+    }
     const vueVersion = Number(`${Vue.version.split('.')[0]}.${Vue.version.split('.')[1]}`)
     if (vueVersion >= 2.5) {
       vm.$_vueTestUtils_scopedSlots = {}
+      vm.$_vueTestUtils_slotScopes = {}
       const renderSlot = vm._renderProxy._t
+
       vm._renderProxy._t = function (name, feedback, props, bindObject) {
         const scopedSlotFn = vm.$_vueTestUtils_scopedSlots[name]
+        const slotScope = vm.$_vueTestUtils_slotScopes[name]
         if (scopedSlotFn) {
           props = { ...bindObject, ...props }
-          vm._renderProxy.props = props
-          return scopedSlotFn.call(vm._renderProxy)
+          const proxy = {}
+          Object.keys(vm._renderProxy).concat(Object.keys(Vue.prototype)).forEach((key) => {
+            if (key[0] === '_') {
+              proxy[key] = vm._renderProxy[key]
+            }
+          })
+          proxy[slotScope] = props
```

Comment:
Since when `slot name`  and  `slot scope` is same,  `vm._renderProxy.[<slot name>]` is changed in `_t` function,
`proxy` is created.

```
<ul>
  <slot name="foo"
    v-for="(item, index) in foo"
    :text="item.text"
    :index="index">
  </slot>
</ul>
```

```
function render() {
  with(this) {
    return _c('ul', [_l((foo), function (item, index) {
      return _t("foo", null, {
        text: item.text,
        index: index
      })
    })], 2)
  }
}
```

scopedSlotFn

```
function anonymous() {
  with(this){return _c('p',{},[_v(_s(foo.index)+","+_s(foo.text))])}
}
```